core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -160,17 +160,16 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     } else false
   }
 
-  private def aclMatch(session: Session, operations: Operation, resource: Resource, principal: KafkaPrincipal, host: String, permissionType: PermissionType, acls: Set[Acl]): Boolean = {
+  private def aclMatch(session: Session, operations: Operation, resource: Resource, principal: KafkaPrincipal, host: String, permissionType: PermissionType, acls: Set[Acl]): Boolean =
     acls.find ( acl =>
```

Comment:
Did you remove the parenthesis in the wrong place? This is the one I was referring to. I think we should probably keep the braces around the method itself (I usually only remove them if it's a single-liner, but that might be my personal preference).